### PR TITLE
Initial PR

### PR DESCRIPTION
**Pull request by jklukas**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+License to be determined.
```
Comment (jklukas) [inline LICENSE]: Should we use an MIT license? Should we attribute this to "Jeff Klukas" or to Simple? Should we move this to the BankSimple org?

File: README.md [hunk 2]
```diff
@@ -2,3 +2,89 @@
...
+Pipelayer uses Data Pipeline's concept of [data staging](http://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/dp-concepts-staging.html) to pull input files from S3 at the beginning of execution and to upload output files back to S3 at the end of execution.
+
+If you follow Pipelayer's directory structure, all of your pipeline logic
+can live in version-controlled flat files. The included command-line interface
+gives you simple commands to validate your pipeline definitions, upload
+task definitions to S3, and activate your pipelines.
+
+## Installation
+
+Pipelayer is (will be...) available from PyPI:
```
Comment (jklukas) [inline README.md]: Do we have a Simple PyPI account? Or should I upload via my personal account?

Comment (jklukas) [inline README.md]: Sounds like Jake's experience is to use personal accounts for clojars, etc. I'll plan to use my own PyPI account.

**Comment by jklukas** [comment]
API docs for this branch are now available at http://pipelayer.readthedocs.org/en/initial/ if that helps anybody in reviewing.

File: docs/source/util.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Pipelayer Util
```
Comment (xstevens) [inline docs/source/util.rst]: s/Pipelayer/Pipewelder?

File: docs/source/core.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Pipelayer Core API
```
Comment (xstevens) [inline docs/source/core.rst]: s/Pipelayer/Pipewelder?

File: pipewelder/cli.py [hunk 5]
```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+The Pipelayer command-line interface.
```
Comment (xstevens) [inline pipewelder/cli.py]: s/Pipelayer/Pipewelder? I'll stop saying this, but you might want to do a case-insensitive recursive grep for the old project name.

**Comment by jklukas** [comment]
I had tried to use an in-place sed invocation to do this finding and replacing. Turns out the `-i` argument on OS X is quite finicky.

Tried again with `find . -type f -print0 | xargs -0 sed -i '' 's,Pipelayer,Pipewelder,g'` and I think I've gotten the rest of them.

**Pull request merged**